### PR TITLE
Issue 1 landing page - Steve Behm 

### PR DESCRIPTION
**Pull request by SteveBehm**
Screen Shot 2022-03-01 at 4 02 48 PM
Screen Shot 2022-03-01 at 4 03 10 PM

### CODE REVIEW

**Comment by SteveBehm** [comment]
https://user-images.githubusercontent.com/96929704/156430791-012667a9-3a43-48bc-972f-770b9563897f.mov


https://user-images.githubusercontent.com/96929704/156430819-59032b74-2569-406a-aca6-9cb133ba7085.mov

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+// when user clicks the try it button it swaps views and produces random image
+$tryItBtn.addEventListener('click', handleClickRandomImage);
+// when user clicks the new button it swaps views and gets a new image
+$newLink.addEventListener('click', handleNewClick);
+// listens for a load event to make a request to the API
+xhr.addEventListener('load', getRandomArtObject);
+// sends the equest to the API
+xhr.send();
```
Comment (codingcodymiller) [inline js/main.js]: You should move these to be with the rest of the methods and properties to set up this request, and then you should put that into a function that you can call.

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  // it is then going to take the primary img url and put it into the src value for
+  // the random image when creating a story.
+  var objectData = new XMLHttpRequest();
+  objectData.open('GET', 'https://collectionapi.metmuseum.org/public/collection/v1/objects/' + randomObjectIdString);
+  objectData.responseType = 'json';
+
+  function getRandomArtImg() {
+    var imgURL = objectData.response.primaryImage;
+    $randomImg.setAttribute('src', imgURL);
+  }
```
Comment (codingcodymiller) [inline js/main.js]: If you give this function a parameter, you can move the definition to the global scope, and just pass `objectData.response.primaryImage` as an argument when you call the function.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (SteveBehm) [reply js/main.js]: What do you mean by you can put that into a function that you can call?

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**